src/header.tsx

```diff
@@ -152,6 +152,10 @@ export type HeaderMenuItemProps = JSX.IntrinsicElements["li"] & {
  */
 export const HeaderMenuItem: React.FC<HeaderMenuItemProps> = ({ icon, children, wrapper, ...rest }) => {
   const config = useAppkitConfig();
+  const focusBgColor = useColorScheme().scheme.includes("high-contrast") ?
+    config.colors.focus : config.colors.neutral10;
+  const focusTextColor = useColorScheme().scheme.includes("high-contrast") ?
+    config.colors.neutral05 : config.colors.neutral90;
```

Comment:
```suggestion
  const focusBgColor = useColorScheme().scheme.includes("high-contrast") 
    ? config.colors.focus 
    : config.colors.neutral10;
  const focusTextColor = useColorScheme().scheme.includes("high-contrast") 
    ? config.colors.neutral05 
    : config.colors.neutral90;
```

Just a formatting thing. That's how it's typically formatted here.